lib/index.js

```diff
@@ -83,21 +99,49 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
             public: true,
           });
       }
-      // TODO: call `brAccount.getAll`
-      // TODO: improve error details
-      throw new BedrockError(
-        'Account does not exist.', 'NotFoundError', {
-          httpStatusCode: 404,
-          public: true,
-        });
+      const {email, cursor = null, limit = 10} = req.query;
+      const {actor} = req.user || {};
+      const query = {'account.email': email};
+      if(cursor) {
+        query['account.id'] = {$gt: cursor};
+      }
+      const fields = {
+        'account': 1,
+        'meta.created': 1,
+        'meta.sequence': 1,
+        'meta.status': 1,
+        'meta.sysResourceRole': 1
+      };
+      const options = {limit, sort: {'account.id': -1}};
+      const records = await brAccount.getAll({actor, query, fields, options});
+      res.json(records);
     }));
 
   app.patch(
     accountPath,
     ensureAuthenticated,
-    asyncHandler(async (req, res, next) => {
-      // TODO: call `brAccount.update`
-      next();
+    intParser(),
+    validate('account.update'),
+    asyncHandler(async (req, res) => {
+      /**
+      * @function accountUpdateHandler
+      * @param {Request} req
+      * @param {Response} res
+      * @param {Function} next
+      * @description patch requests to /:account result in an updated account
+      * patches need to be in the:
+      * [json patch]{@link https://www.npmjs.com/package/fast-json-patch}
```

Comment:
https://tools.ietf.org/html/rfc6902 <-- RFC 6902 is a better reference than the particular implemention we're using today.

Reply:
can I link to both?